zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/handlers/UserTaskCompletionVariableHandler.java

```diff
@@ -31,7 +31,7 @@ public class UserTaskCompletionVariableHandler
   private static final Logger LOGGER =
       LoggerFactory.getLogger(UserTaskCompletionVariableHandler.class);
 
-  private static final String ID_PATTERN = "%s-%s";
+  private static final String ID_PATTERN = "%s-%s" + TaskTemplate.LOCAL_VARIABLE_SUFFIX;
```

Comment:
❓ I don't understand why this handler is required? Snapshotting of variables happens in v1, but we no longer will do this with v2. IMO, this handler can be removed. (If we come to that conclusion, we should do this in a separate PR and it does not block this PR.)

Reply:
After some consideration, this handler must be rewritten to support the Tasklist v1 API. I will do this separately. This is not necessary for the v2 API.

Reply:
@romansmirnov I believe we still need this snapshot of User Task completion variables for Form inputs and variables submitted via the User Task. These variables should maintain the state that they had when they were initially submitted by the end-user.